tests/src/Contact/FriendSuggest/Factory/FriendSuggestTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Friendica\Test\src\Contact\FriendSuggest\Factory;
+
+use Friendica\Contact\FriendSuggest\Factory\FriendSuggest;
+use Friendica\Contact\FriendSuggest\Entity;
+use Friendica\Test\MockedTest;
+use Friendica\Util\Logger\VoidLogger;
+
+class FriendSuggestTest extends MockedTest
+{
+	public function dataCreate()
+	{
+		return [
+			'default' => [
+				'input' => [
+					'uid'     => 12,
+					'cid'     => 13,
+					'name'    => 'test',
+					'url'     => 'https://friendica.local/profile/test',
+					'request' => 'https://friendica.local/dfrn_request/test',
+					'photo'   => 'https://friendica.local/photo/profile/test',
+					'note'    => 'a common note',
+					'created' => '2021-10-12 12:23:00'
+				],
+				'assertion' => [
+					'uid'     => 12,
+					'cid'     => 13,
+					'name'    => 'test',
+					'url'     => 'https://friendica.local/profile/test',
+					'request' => 'https://friendica.local/dfrn_request/test',
+					'photo'   => 'https://friendica.local/photo/profile/test',
+					'note'    => 'a common note',
+					'created' => new \DateTime('2021-10-12 12:23:00', new \DateTimeZone('UTC')),
+					'id'      => null,
+				],
+			],
+			'minimum' => [
+				'input' => [
+					'id' => 20,
+				],
+				'assertion' => [
+					'id' => 20,
+				]
+			],
+			'full' => [
+				'input' => [
+					'uid'     => 12,
+					'cid'     => 13,
+					'name'    => 'test',
+					'url'     => 'https://friendica.local/profile/test',
+					'request' => 'https://friendica.local/dfrn_request/test',
+					'photo'   => 'https://friendica.local/photo/profile/test',
+					'note'    => 'a common note',
+					'created' => '2021-10-12 12:23:00',
+					'id'      => 666,
+				],
+				'assertion' => [
+					'uid'     => 12,
+					'cid'     => 13,
+					'name'    => 'test',
+					'url'     => 'https://friendica.local/profile/test',
+					'request' => 'https://friendica.local/dfrn_request/test',
+					'photo'   => 'https://friendica.local/photo/profile/test',
+					'note'    => 'a common note',
+					'created' => new \DateTime('2021-10-12 12:23:00', new \DateTimeZone('UTC')),
+					'id'      => 666,
+				],
+			],
+		];
+	}
+
+	public function assertFriendSuggest(Entity\FriendSuggest $friendSuggest, array $assertion)
```

Comment:
Traditionally in `assert*` method, the assertion value is the first parameter. You can see it in the following `assertEquals` calls in this method.
```suggestion
	public function assertFriendSuggest(array $assertion, Entity\FriendSuggest $friendSuggest)
```